cms/templates/cms/program_page.html

```diff
@@ -6,6 +6,20 @@
 
 {% block title %}{% trans "MIT Micromasters" %}{% endblock %}
```

Comment:
This is outside the PR scope but can you update `Micromasters` to `MicroMasters` here too?